### PR TITLE
Chore(docs): Updated README

### PR DESCRIPTION
**Pull request by alithethird**
Updated README file to the latest standard.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Manging Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
...
-* [Get started](https://charmhub.io/discourse-k8s/docs/getting-started)
+## Integrations
 
-Thinking about using the Discourse Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
+This charm can be integrated with other Juju charms and services:
 
----
+    - [Redis](https://charmhub.io/redis-k8s): Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker.
+    - [Saml](https://charmhub.io/saml-integrator): SAML is an open standard used for authentication.
+    - [Postgresql](https://charmhub.io/postgresql-k8s): PostgreSQL is a powerful, open source object-relational database system. It has more than 15 years of active development and a proven architecture that has earned it a strong reputation for reliability, data integrity, and correctness.
```
Comment (erinecon) [inline README.md:45]: These lines are rendering as code -- maybe because of how they're indented?

For the PostgreSQL description, I recommend that you remove the second sentence.

File: README.md [hunk 2]
```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
...
 
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/discourse-k8s/docs).
+    and much more. You can find the full list of integrations [here](https://charmhub.io/discourse-k8s/integrations).
```
Comment (erinecon) [inline README.md:47]: I would rephrase these two sentences to say:
> `For a full list of integrations, see the [Charmhub documentation](https://charmhub.io/discourse-k8s/integrations).`

File: README.md [hunk 3]
```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
...
+
+It allows for deployment on
+many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
 [Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
 offerings.
 
-As such, the charm makes it easy for those looking to take control of their own
-discussion platform whilst keeping operations simple, and gives them the
-freedom to deploy on the Kubernetes platform of their choice.
+For information about how to deploy, integrate, and manage this charm,
```
Comment (erinecon) [inline README.md:23]: Nitpick (to adhere to style guide): Remove comma between after "integrate", in other words:
> For information about how to deploy, integrate and manage this charm...

File: README.md [hunk 4]
```diff
@@ -9,36 +9,50 @@ A juju charm deploying and managing Discourse on Kubernetes. Discourse is the
...
 
-## Project and community
+You can follow the tutorial [here](https://charmhub.io/discourse-k8s/docs/tutorial).
+
+### Basic operations
+
+The following actions are available for this charm:
+    - anonymize-user: anonymize a user
+    - create-user: create a new user
+    - promote-user: promote a user to admin
```
Comment (erinecon) [inline README.md:35]: These lines aren't rendering as a bulleted list for me -- maybe because of how they're indented?

Another nitpick: I think it may look better to format the actions as code, in other words:

- `anonymize-user`: anonymize a user
- `create-user`: create a new user
- `promote-user`: promote a user to admin

**Review by erinecon — APPROVED** [review]
Thanks for updating the README -- it looks good to me!

**Review by amandahla — APPROVED** [review]
